west.yml

```diff
@@ -79,6 +79,12 @@ manifest:
       path: modules/hal/gigadevice
       groups:
         - hal
+    - name: hal_hpmicro
+      url: https://github.com/hpmicro/zephyr_hal_hpmicro.git
+      revision: 83806f293cf2709e84f4649423c9729b2f507766
+      path: modules/hal/hpmicro
+      groups:
+        - hal
```

Comment:
https://github.com/zephyrproject-rtos/hal_hpmicro has been created as per https://github.com/zephyrproject-rtos/zephyr/issues/46301.

Please update the manifest to reference the above repository.

Reply:
> https://github.com/zephyrproject-rtos/hal_hpmicro has been created as per #46301.
> 
> Please update the manifest to reference the above repository.

OK, it has been updated. Please start the next step as soon as possible.